README.md

```diff
@@ -13,16 +13,18 @@ var cleankill = require('cleankill');
 To register a handler:
 
 ```js
-cleankill.onInterrupt(function(done) {
-  // do things.
-  done();
+cleankill.onInterrupt(() => {
```

Comment:
For the record I'm also against one-line `if` statements, `for` statements, `while` loops, etc (and so is just about every google style guide).